optimum/onnx/auto/configuration_onnx_auto.py

```diff
@@ -0,0 +1,211 @@
+# coding=utf-8
+# Copyright 2018 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Auto ONNX Config class."""
+import importlib
+import re
+import warnings
+from collections import OrderedDict
+from typing import List, Union
+
+from transformers.configuration_utils import PretrainedConfig
+from transformers.file_utils import CONFIG_NAME
+from transformers.models.auto.configuration_auto import (
+    CONFIG_MAPPING,
+    MODEL_NAMES_MAPPING,
+    AutoConfig,
+    _get_class_name,
+    _LazyConfigMapping,
+)
+from transformers.models.auto.dynamic import get_class_from_dynamic_module
+from transformers.onnx.config import OnnxConfig
+from transformers.utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+ONNX_CONFIG_MAPPING_NAMES = OrderedDict(
```

Comment:
Why do we need an `OrderedDict` here?

Reply:
> Why do we need an `OrderedDict` here?

There are models in transformers with OnnxConfig defined, but some are not. So here I track all those models with their OnnxConfig defined.

Reply:
But indeed, it will be better to get it directly from FeaturesManager so that we don't need to change it all the time.